orbit/pkg/table/dataflattentable/exec.go

```diff
@@ -35,11 +34,11 @@ func WithBinDirs(binDirs ...string) ExecTableOpt {
 	}
 }
 
-func TablePluginExec(logger log.Logger, tableName string, dataSourceType DataSourceType, execArgs []string, opts ...ExecTableOpt) *table.Plugin {
+func TablePluginExec(tableName string, dataSourceType DataSourceType, execArgs []string, opts ...ExecTableOpt) *table.Plugin {
```

Comment:
Why do we remove the logger argument and use the global logger instead of replacing it with a `zerolog` logger like the other tables here?

Also I cannot find the corresponding callsite for this function. Is it used at all?

Reply:
Aha good catch! I thought I'd added them all back.

It doesn't appear to be used, but I would rather leave it for now and we can remove later in a different ticket.